apollo-backend/pom.xml

```diff
@@ -185,10 +185,21 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>16.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <version>1.15.1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
why do we need this dependency?
and if we need it why do we use different versions 1.15.1 and 1.15.2?

Reply:
@matvey-mtn I changed to 1.15.1 both - but we need all the dependencies else it won't work.